### PR TITLE
feature(re-design UI for sending invite to other online players)

### PR DESCRIPTION
**Pull request by Orlayhemmy**
#### What does this PR do?
It re-designs the UI for sending invites to online users

#### Description of Task to be completed?
- add tabs to hold players in the game and online users
- add notifications on all pages navbar
- fix multiplicating players
- fix abandon game error

#### What are the relevant pivotal tracker stories?
Story Id #157610021

#### Screenshots (if appropriate)
screen

screen shot 2018-05-16 at 5 09 30 pm

### CODE REVIEW

File: public/js/controllers/index.js [hunk 1]
```diff
@@ -166,10 +178,23 @@ angular.module('mean.system').controller('IndexController', [
       messageArray.push(message);
       $scope.notifications = messageArray;
       $scope.messageLength = messageArray.length;
+      console.log('@@@@@@@@@@@',messageArray);
     });
 
-    window.onload = () => {
-      $scope.userGames()
+    $scope.tab2 = false;
```
Comment (chuzksy-codeactive) [inline public/js/controllers/index.js]: I know you have put in much work to make this feature work. Good job. But next time, try to use an appropriate variable/property names. Using names like tab1 and tab2 does not convey much to the reader. Names like playerTab, sendInvitesTab would be more appropriate to use. Good job. Keep it up

File: config/socket/socket.js [hunk 2]
```diff
@@ -69,31 +69,42 @@ module.exports = function (io) {
         username: data
       };
       onlineUsers.push(user);
-      io.sockets.emit('people', onlineUsers)
-
+      io.sockets.emit('people', onlineUsers);
+      console.log('connect', onlineUsers);
```
Comment (amarachukwu-agbo) [inline config/socket/socket.js]: Please remove the console statement here

File: public/js/controllers/index.js [hunk 3]
```diff
@@ -166,10 +178,23 @@ angular.module('mean.system').controller('IndexController', [
       messageArray.push(message);
       $scope.notifications = messageArray;
       $scope.messageLength = messageArray.length;
+      console.log('@@@@@@@@@@@',messageArray);
```
Comment (amarachukwu-agbo) [inline public/js/controllers/index.js]: Here too. Please remove console statement.

File: public/js/controllers/index.js (hunk 1 above)
Comment (Orlayhemmy) [reply public/js/controllers/index.js]: I have done that, thanks

File: config/socket/socket.js (hunk 2 above)
Comment (Orlayhemmy) [reply config/socket/socket.js]: I have done that, thanks

File: config/socket/socket.js [hunk 4]
```diff
@@ -69,31 +69,42 @@ module.exports = function (io) {
...
+          .userId;
+        if (socketId !== undefined) {
+          io
+            .to(socketId)
+            .emit('invitation', {
+              html: `You have been Invited to play a game.<br/> Click on this <span class="invite-link"><a style="color: red;" href=${
+                data.gameLink
+              } target="_blank">link to join</a></span>`
+            });
+            console.log(data)
```
Comment (andela-moe) [inline config/socket/socket.js]: Is this for development purposes? I believe you can take this out of your code.

File: public/js/controllers/index.js [hunk 5]
```diff
@@ -166,10 +178,23 @@ angular.module('mean.system').controller('IndexController', [
...
     });
 
-    window.onload = () => {
-      $scope.userGames()
+    $scope.tab2 = false;
+    $scope.tab1 = true;
+    $scope.inviteModal = false;
+    $scope.showtab1 = () => {
```
Comment (andela-moe) [inline public/js/controllers/index.js]: The feedback by @chuzksy-codeactive should also apply here. These names (_showtab1_, _showtab2_) are not descriptive enough based on their essence. I suggest you use variable/parameter names that are descriptive.

File: public/views/app.html [hunk 6]
```diff
@@ -97,18 +113,44 @@ <h5 >
...
-        </button>
-      </form> 
-      <div player style="width:100%">
-
+    <aside class="players">
+      <div class="row" ng-controller="IndexController">
+        <div class="col s12">
+          <ul class="tabs">
+            <li class="tab">
+              <a class="active" href="#" ng-click="showtab1()">Players</a>
```
Comment (andela-moe) [inline public/views/app.html]: I believe these (`showtab1()`, `showtab2()`) should also be corrected based on feedback you had received.

**Review by andela-moe — COMMENTED** [review]
Please review and resolve feedback

**Comment by andela-moe** [comment]
Closing this now. Please re-open when you are done implementing feedback.

File: public/js/controllers/index.js (hunk 5 above)
Comment (Orlayhemmy) [reply public/js/controllers/index.js]: I have implemented it

File: public/views/app.html (hunk 6 above)
Comment (Orlayhemmy) [reply public/views/app.html]: I have implemented it

File: config/socket/socket.js (hunk 4 above)
Comment (Orlayhemmy) [reply config/socket/socket.js]: I have removed it. Thanks

File: public/js/controllers/index.js (hunk 3 above)
Comment (Orlayhemmy) [reply public/js/controllers/index.js]: I have done that

**Pull request closed without merging**